### PR TITLE
lmao

### PR DESCRIPTION
**Pull request by MCer4294967296**
I have these two piece of artifact that my uncle gave me, he says it's from my ancestors dated way back in time, probably around 999 BCE. Could you please help me identify them? It would mean so much to my family. I will repay you with my nonexistent Crazy Thursday coupon.

Do be careful with them though: they both seem dangerous.

### CODE REVIEW

**Comment by MCer4294967296** [comment]
Wait will this violate my company's CLA? If it got requested to be taken down I'll have to take these back, please don't hate me.

File: src/artifacts/shutdown.keyboard.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: Shutdown
+language: ?
+status: OnExhibit
+tags: ["Dangerous"]
+hall: HallOfTroll
+code: |-
+  Alt + F4
+desc: |-
+  Don't try this at home!
+  This, with pretty high probably and confidence, kills your window.
```
Comment (YilunAllenChen) [inline src/artifacts/shutdown.keyboard.yaml:10]: We don't support this language yet

File: src/artifacts/bigmalloc.c.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+title: Infinite Memory
+language: C
+status: OnExhibit
+tags: ["Dangerous"]
+hall: HallOfTroll
+code: |-
+  \#include <malloc.h>
```
Comment (YilunAllenChen) [inline src/artifacts/bigmalloc.c.yaml:7]: I don't think we need the backslash?

File: src/artifacts/bigmalloc.c.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+title: Infinite Memory
```
Comment (YilunAllenChen) [inline src/artifacts/bigmalloc.c.yaml:1]: more like infinite malloc

**Comment by YilunAllenChen** [comment]
Er I think it’s fine as long as:
1. You didn’t do this on your company laptop / using your company GitHub
account
2. You didn’t submit company code (IP)
3. The things you do conflicts with your company’s interests

So I think we’re good?

If you feel unsure feel free to close, always safe than sorry 👾

**Comment by YilunAllenChen** [comment]
(closing due to inactivity, happy to reopen)

**Pull request closed without merging**